### DISCUSSION TITLE
Creality Ender 5 Pro (2019)

### DISCUSSION THREAD

**Discussion by TonyK54** [OrcaSlicer/OrcaSlicer#832] [Ideas]
It would be so good if OrcaSlicer supported my Ender 5 Pro from 2019 (still original 8-bit board). It has OctoPrint over WiFi.

I've got v1.6.1 of the source code and have updated the JSON files and added the image file. Tested locally and it is selectable and it connects to the Pi without issue.

Still need to slice. However the JSON files were copied from the Ender-3 files and the biggest difference between an Ender-3 and Ender-5 is the height available. The Ender-3 is 250mm and the Ender-5 and Ender-5 Pro from 2019 is 300mm.

What is the best way to submit my changes? I think I'm asking what branch is used: master or SoftFever? 

I use GitHub professionally and am using GitHub Desktop. Creating a pull request will not be an issue, just want to know where to put the files. 

Thank you,

**Comment by TonyK54** [comment]
Answered my own question by reviewing the existing pull requests. My own pull request is at https://github.com/SoftFever/OrcaSlicer/pull/839 and needs review and merging please @SoftFever . Thank you,

**Reply by SoftFever** [reply]
Merged. thank you!
